### DISCUSSION TITLE
Assign one or multiple orders to an indicated waiter

### DISCUSSION THREAD

**Discussion by boudo2017** [Blair2004/NexoPOS#481] [Ideas]
Hi,
In these days if you order by phone/take away ,the cashier inform you : you will be served by waiter N 1, 2, 3. ,name.
when you come you will be asked security men or receptionist your waiter number and name? you will pay this waiter and you will get your bill.(actually in almost of cafeteria)
the same thing can be done for dine-in and delivery orders.
Could you , add this feature?
regards

**Comment by Blair2004** [comment]
That needs seems specific to a restaurant need and therefore is not general. I rather see a way to assign a table to a waiter. But this depends on wether people upvote this. We'll keep it aside.
